### PR TITLE
No mouse grab

### PR DESCRIPTION
**Pull request by akallabeth**
https://github.com/FreeRDP/FreeRDP/pull/9099

### CODE REVIEW

**Comment by akallabeth** [comment]
@freerdp-bot test please

**Comment by akallabeth** [comment]
@freerdp-bot come on, please test

File: client/X11/xf_graphics.c [hunk 1]
```diff
@@ -562,6 +562,11 @@ static BOOL xf_Pointer_SetPosition(rdpContext* context, UINT32 x, UINT32 y)
 	WLog_DBG(TAG, "%s: %" PRIu32 "x%" PRIu32, __func__, x, y);
 	if (xfc->remote_app && !xfc->focused)
 		return TRUE;
+	else if (!xfc->remote_app)
+	{
+		if (!xfc->focused)
```
Comment (pnowack) [inline client/X11/xf_graphics.c:567]: Like in #9099 combine this condition with the one above to just `if (!xfc->focused)`

**Review by pnowack — APPROVED** [review]

File: client/X11/xf_graphics.c (hunk 1 above)
Comment (akallabeth) [reply client/X11/xf_graphics.c]: updated from upstream pr

**Pull request merged**